src/models/parallelCoordinates.js

```diff
@@ -369,13 +371,34 @@ nv.models.parallelCoordinates = function() {
                         f.hasOnlyNaN = true;
                 });
                 dispatch.brushEnd(active, hasActiveBrush);
+            }           
+            function updateTicks() {
```

Comment:
Ticks visible are min and max for each axis, if there is a brush, brush extent is display.